email/1.6/modules/ROOT/pages/email-move-folder.adoc

```diff
@@ -0,0 +1,163 @@
+= Move Messages Between Folders for IMAP/S Connection Providers Examples - Mule 4
+
+Anypoint Connector for Email (Email Connector) provides the *Move to folder* operation that enables you to move messages to folders for IMAP and IMAPS connection providers. The operation's main features include:
+
+* Moving messages between folders of the email server.
+* Preserving the moved message UID.
+* Creating the target folder if the folder does not exist or fails depending on a parameter.
+
+The following examples illustrate how to configure the operation for the connector.
+
+== Move a Message from One Folder to Another Folder
+
+The following example illustrates how to move a received message from one folder to another folder. An *On New Email - IMAP* source initiates the flow, and then the *Mark as read* operation reads the received email in the `INBOX` folder. Subsequently, the *Move to folder* operation moves the email to the `TARGET` folder:
+
+The following screenshot shows the app flow for this example:
+
+image::email-movetofolder-flow-1.png[Move a Message from One Folder to Another Folder flow]
+
+To create the flow: +
+
+. Create a new Mule project in Studio.
+. In the *Mule Palette* view, drag the *On New Email - IMAP* operation onto the canvas.
+. On the *On New Email - IMAP* configuration screen, click the plus sign (*+*) next to the *Connector configuration* field to configure a global element for the operation.
+. Set *Connection* to *IMAPS Connection*.
+. Complete the following fields:
++
+* *Host*: `outlook.office365.com`
```

Comment:
I'd prefer to use another target example server to avoid confusions. We already provide an Office 365 connector.